pkg/privilege/privileges/cache.go

```diff
@@ -1647,9 +1857,9 @@ func (h *Handle) Get() *MySQLPrivilege {
 }
 
 // Update loads all the privilege info from kv storage.
-func (h *Handle) Update(ctx sessionctx.Context) error {
+func (h *Handle) Update() error {
 	var priv MySQLPrivilege
-	err := priv.LoadAll(ctx)
+	err := priv.LoadAll(h.sctx)
```

Comment:
So this PR still loads all users into memory and partial load will be implemented in other PR?

Reply:
Yes, this is the first step and the following PR will handle the remains.